libcontainer/intelrdt/intelrdt.go

```diff
@@ -504,6 +522,20 @@ func (m *Manager) GetPath() string {
 	return m.path
 }
 
+// GetMonPath returns path of the monitoring group of the container. Returns an
+// empty string if the container does not have a individual dedicated
+// monitoring group.
+func (m *Manager) GetMonPath() string {
+	if closPath := m.GetPath(); closPath != "" && m.config.IntelRdt.EnableMonitoring {
+		path, err := securejoin.SecureJoin(filepath.Join(closPath, "mon_groups"), m.id)
```

Comment:
This assumes m.id can contain unsafe elements like "../", it looks like this can't be the case since `id` is container ID, which we validate in `validateID` and reject such stuff.

So, this might be simplified to just `path.Join` (or `filepath.Join`), or even `closPath + "/mon_groups/" + m.id`

Reply:
Check. I changed this back to filepath.Join